staging/src/k8s.io/apiserver/pkg/util/openapi/proto.go

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
90% of the logic for this is from https://github.com/kubernetes/kubectl/blob/master/pkg/framework/openapi/openapi.go